AsyncDisplayKit/ASCellNode.m

```diff
@@ -111,7 +111,7 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
   static const CGFloat kHorizontalPadding = 15.0f;
   static const CGFloat kVerticalPadding = 11.0f;
   UIEdgeInsets insets = UIEdgeInsetsMake(kVerticalPadding, kHorizontalPadding, kVerticalPadding, kHorizontalPadding);
-  return [ASInsetLayoutSpec newWithInsets:insets child:_textNode];
+  return [ASInsetLayoutSpec insetLayoutSpecWithInsets:insets child:_textNode];
```

Comment:
Thanks for the new names!  Definitely the right call before these APIs go out in a stable release.